locals.tf

```diff
@@ -5,33 +5,48 @@ locals {
     lookup(module.stage_label.descriptors, var.descriptor_name, module.stage_label.id), "/${module.stage_label.delimiter}${module.stage_label.delimiter}+/", module.stage_label.delimiter
   ), module.stage_label.delimiter) : null
 
+  create_default_roles     = module.this.enabled && var.create_default_roles
   schema_object_stage_name = "\"${one(snowflake_stage.this[*].database)}\".\"${one(snowflake_stage.this[*].schema)}\".\"${one(snowflake_stage.this[*].name)}\""
```

Comment:
Please add an if-clause here, when `module.this.enabled ? ... : null` just to be sure